react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/IllegalSelectionIndexException.java

```diff
@@ -0,0 +1,47 @@
+package org.wordpress.mobile.ReactNativeAztec;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+
+import org.jetbrains.annotations.NotNull;
+import org.wordpress.aztec.spans.UnknownHtmlSpan;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class IllegalSelectionIndexException extends Exception {
+    IllegalSelectionIndexException(int selectionStart, int selectionEnd, int textLength, ReactAztecText view) {
+        super(createMessage(selectionStart, selectionEnd, textLength, getHtmlTags(view, textLength)));
+    }
+
+    private static String createMessage(int selectionStart, int selectionEnd, int textLength, List<String> tags) {
```

Comment:
Looks like throughout the class, it's the _unknown_ html tags that get wrangled so, wdyt about renaming the variables to something like `unknownHtmlTags`? `createMessage` itself prepends the tags list with the `UnknownHtmlSpan`'s classes name, already assuming that the tags list is only of unknown html tags.

Reply:
That is a much more precise name. Updated in 32d722f!